index.js

```diff
@@ -32,6 +32,24 @@ module.exports = {
         isInternal: true,
         manifest: safejs.nfs.manifest,
         methods: safejs.nfs
+    },
+    {
+        name: 'structuredData',
```

Comment:
I'll keep these as is for now, but I'd like to move towards something like `safeStructuredData` for consistency/clarity.